src/layout/index.tsx

```diff
@@ -20,6 +20,7 @@ const StyledContent = styled(Content)`
 `
 const StyledBody = styled.div`
   padding: 24px;
+  padding-top: 0;
```

Comment:
@shootermv this comment can be [a suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) :)
```suggestion
  padding: 0 24px;
```